haystack/document_store/milvus2x.py

```diff
@@ -0,0 +1,691 @@
+import logging
+from typing import Any, Dict, Generator, List, Optional, Union
+
+import numpy
+import numpy as np
+
+from scipy.special import expit
+from tqdm import tqdm
+
+from haystack import Document
+from haystack.document_store.sql import SQLDocumentStore
+from haystack.retriever.base import BaseRetriever
+from haystack.utils import get_batches_from_generator
+
+logger = logging.getLogger(__name__)
+
+
+class Milvus2DocumentStore(SQLDocumentStore):
+    """
+    Milvus (https://milvus.io/) is a highly reliable, scalable Document Store specialized on storing and processing vectors.
```

Comment:
Can you please add a few lines here about:
- the "experimental" stage of the implementation
-  the differences to milvus 1.0 (why users should consider upgrading? )
- current limitations (e.g. delete + update_embeddings won't work)
- prerequisites ( pip install pymilvus===2.0.0rc6)